### PR TITLE
Upgrade addon via `ember init`

### PR DESCRIPTION
**Pull request by elwayman02**
Remove unnecessary dependencies
Move bower deps to devDep
Remove ember-paper import from `app/`, got installed accidentally from the blueprint and should only be used for dummy app

### CODE REVIEW

**Comment by elwayman02** [comment]
It looks like NPM might be barfing in travis atm, doesn't seem to have anything to do with the PR

File: tests/dummy/app/templates/application.hbs [hunk 1]
```diff
@@ -1,77 +1,3 @@
-{{#paper-nav-container}}
+<h2 id="title">Welcome to Ember</h2>
 
-  {{#paper-sidenav class='md-sidenav-left md-whiteframe-z2' locked-open='gt-sm'}}
```
Comment (sir-dunxalot) [inline tests/dummy/app/templates/application.hbs]: Looks like this file got accidentally overwritten.

**Comment by sir-dunxalot** [comment]
Looks awesome - thanks for putting the time into this. Once the `dummy/templates/application.hbs` file is restored I'll merge and release.

File: tests/dummy/app/templates/application.hbs (hunk 1 above)
Comment (elwayman02) [inline tests/dummy/app/templates/application.hbs]: Whoops, sorry! :)

**Comment by elwayman02** [comment]
@sir-dunxalot fixed! :)

**Comment by sir-dunxalot** [comment]
You the man! Merging and releasing now.

**Pull request merged**